.github/agents/planner.agent.md

```
@@ -0,0 +1,169 @@
+---
+name: Planner
+description: Research codebase context and return structured findings for planning multi-phase development tasks
+argument-hint: Research goal or problem statement
+target: vscode
+tools: ['search', 'edit', 'usages', 'problems', 'changes', 'testFailure', 'githubRepo', 'github/*', 'mslearn/*', 'context7/*', 'angular/*']
+model: Claude Sonnet 4.5
+---
+
+---
+name: OrchestraPlanner
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)